gensim/models/wrappers/fasttext.py

```diff
@@ -286,7 +286,7 @@ def load_dict(self, file_handle, encoding='utf8'):
         # Vocab stored by [Dictionary::save](https://github.com/facebookresearch/fastText/blob/master/src/dictionary.cc)
         assert len(self.wv.vocab) == nwords, 'mismatch between vocab sizes'
         assert len(self.wv.vocab) == vocab_size, 'mismatch between vocab sizes'
-        ntokens = self.struct_unpack(file_handle, '@1q')
+        self.struct_unpack(file_handle, '@1q') # number of tokens 
```

Comment:
PEP8: Inline comments should be separated by at least two spaces from the statement